composer.json

```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "cboden/ratchet": "0.3.*",
-        "doctrine/orm": "2.4.*",
+        "doctrine/orm": "2.5.*",
```

Comment:
@turbo-ele ... we need to update doctrine, please coordinate with @danrot /cc @christianbader